src/links.ts

```diff
@@ -45,6 +56,21 @@ export function getLinks(source: string, baseUrl: string): string[] {
 
 function parseAttr(name: string, value: string): string[] {
   switch (name) {
+    case 'style':
+      return value
+        .split(';')
+        .filter(Boolean)
+        .map((rules: string) => {
+          const [attr, value] = rules.split(':');
```

Comment:
I'm worried about this parser being a little naive.  For example:

```css
h4, h1 a:hover {
	text-decoration: none;
	color: #333;
}
```

Wouldn't the `:` in the `a:hover` cause some trouble in here?  I don't have an answer quite yet, but I'm wondering if we should look into a real CSS lexer/parser similar to cheerio.

Reply:
Absolutely. Maybe PostCSS parser?

Reply:
Some note. This code is about style attribute not style tag so right here we have no problem with pseude-class & pseudo-elements (event with regexp).

Reply:
Feature is extracted to https://github.com/JustinBeckwith/linkinator/pull/113